<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PokedexSecondPage</class>
 <widget class="QDialog" name="PokedexSecondPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1160</width>
    <height>774</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Pokedex</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1231</width>
     <height>771</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="1">
     <layout class="QVBoxLayout" name="verticalLayout2_2">
      <item>
       <widget class="QLabel" name="Pokemon16">
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 11pt &quot;Fixedsys&quot;;
color: rgb(0, 0, 0);
</string>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="CheckButton16">
        <property name="styleSheet">
         <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0.972, y2:0.909, stop:0 rgba(220, 0, 0, 253), stop:0.301676 rgba(147, 0, 0, 254), stop:0.653631 rgba(131, 0, 0, 254), stop:1 rgba(0, 0, 0, 255));

font: 11pt &quot;Fixedsys&quot;;
border-color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>Check</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="1">
     <layout class="QVBoxLayout" name="verticalLayout6_2">
      <item>
       <widget class="QLabel" name="Pokemon20">
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 11pt &quot;Fixedsys&quot;;
color: rgb(0, 0, 0);
</string>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="CheckButton20">
        <property name="styleSheet">
         <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0.972, y2:0.909, stop:0 rgba(220, 0, 0, 253), stop:0.301676 rgba(147, 0, 0, 254), stop:0.653631 rgba(131, 0, 0, 254), stop:1 rgba(0, 0, 0, 255));

font: 11pt &quot;Fixedsys&quot;;
border-color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>Check</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QVBoxLayout" name="verticalLayout5_2">
      <item>
       <widget class="QLabel" name="Pokemon19">
        <property name="styleSheet">
         <string notr="true">font: 11pt &quot;Fixedsys&quot;;
color: rgb(0, 0, 0);
</string>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="CheckButton19">
        <property name="styleSheet">
         <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0.972, y2:0.909, stop:0 rgba(220, 0, 0, 253), stop:0.301676 rgba(147, 0, 0, 254), stop:0.653631 rgba(131, 0, 0, 254), stop:1 rgba(0, 0, 0, 255));

font: 11pt &quot;Fixedsys&quot;;
border-color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>Check</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout1_2">
      <item>
       <widget class="QLabel" name="Pokemon15">
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 11pt &quot;Fixedsys&quot;;
color: rgb(0, 0, 0);
</string>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="CheckButton15">
        <property name="styleSheet">
         <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0.972, y2:0.909, stop:0 rgba(220, 0, 0, 253), stop:0.301676 rgba(147, 0, 0, 254), stop:0.653631 rgba(131, 0, 0, 254), stop:1 rgba(0, 0, 0, 255));

font: 11pt &quot;Fixedsys&quot;;
border-color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>Check</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="3" column="0">
     <layout class="QVBoxLayout" name="verticalLayout9_2">
      <item>
       <widget class="QLabel" name="Pokemon23">
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 11pt &quot;Fixedsys&quot;;
color: rgb(0, 0, 0);
</string>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="CheckButton23">
        <property name="styleSheet">
         <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0.972, y2:0.909, stop:0 rgba(220, 0, 0, 253), stop:0.301676 rgba(147, 0, 0, 254), stop:0.653631 rgba(131, 0, 0, 254), stop:1 rgba(0, 0, 0, 255));

font: 11pt &quot;Fixedsys&quot;;
border-color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>Check</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="3">
     <layout class="QVBoxLayout" name="verticalLayout_9">
      <item>
       <widget class="QLabel" name="Pokemon22">
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 11pt &quot;Fixedsys&quot;;
color: rgb(0, 0, 0);
</string>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="CheckButton22">
        <property name="styleSheet">
         <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0.972, y2:0.909, stop:0 rgba(220, 0, 0, 253), stop:0.301676 rgba(147, 0, 0, 254), stop:0.653631 rgba(131, 0, 0, 254), stop:1 rgba(0, 0, 0, 255));

font: 11pt &quot;Fixedsys&quot;;
border-color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>Check</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="5" column="0">
     <layout class="QVBoxLayout" name="verticalLayout13_2">
      <item>
       <widget class="QLabel" name="emptyLabel">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="BackButton">
        <property name="styleSheet">
         <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0.972, y2:0.909, stop:0 rgba(220, 0, 0, 253), stop:0.301676 rgba(147, 0, 0, 254), stop:0.653631 rgba(131, 0, 0, 254), stop:1 rgba(0, 0, 0, 255));

font: 11pt &quot;Fixedsys&quot;;
border-color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>Return</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="2">
     <layout class="QVBoxLayout" name="verticalLayout7_2">
      <item>
       <widget class="QLabel" name="Pokemon21">
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 11pt &quot;Fixedsys&quot;;
color: rgb(0, 0, 0);
</string>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="CheckButton21">
        <property name="styleSheet">
         <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0.972, y2:0.909, stop:0 rgba(220, 0, 0, 253), stop:0.301676 rgba(147, 0, 0, 254), stop:0.653631 rgba(131, 0, 0, 254), stop:1 rgba(0, 0, 0, 255));

font: 11pt &quot;Fixedsys&quot;;
border-color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>Check</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="3" column="2">
     <layout class="QVBoxLayout" name="verticalLayout11_2">
      <item>
       <widget class="QLabel" name="Pokemon25">
        <property name="styleSheet">
         <string notr="true">font: 11pt &quot;Fixedsys&quot;;
color: rgb(0, 0, 0);
</string>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="CheckButton25">
        <property name="styleSheet">
         <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0.972, y2:0.909, stop:0 rgba(220, 0, 0, 253), stop:0.301676 rgba(147, 0, 0, 254), stop:0.653631 rgba(131, 0, 0, 254), stop:1 rgba(0, 0, 0, 255));

font: 11pt &quot;Fixedsys&quot;;
border-color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>Check</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="3" column="1">
     <layout class="QVBoxLayout" name="verticalLayout10_2">
      <item>
       <widget class="QLabel" name="Pokemon24">
        <property name="styleSheet">
         <string notr="true">font: 11pt &quot;Fixedsys&quot;;
color: rgb(0, 0, 0);
</string>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="CheckButton24">
        <property name="styleSheet">
         <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0.972, y2:0.909, stop:0 rgba(220, 0, 0, 253), stop:0.301676 rgba(147, 0, 0, 254), stop:0.653631 rgba(131, 0, 0, 254), stop:1 rgba(0, 0, 0, 255));

font: 11pt &quot;Fixedsys&quot;;
border-color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>Check</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="2">
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QLabel" name="Pokemon17">
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 11pt &quot;Fixedsys&quot;;
color: rgb(0, 0, 0);
</string>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="CheckButton17">
        <property name="styleSheet">
         <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0.972, y2:0.909, stop:0 rgba(220, 0, 0, 253), stop:0.301676 rgba(147, 0, 0, 254), stop:0.653631 rgba(131, 0, 0, 254), stop:1 rgba(0, 0, 0, 255));

font: 11pt &quot;Fixedsys&quot;;
border-color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>Check</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="3" column="3">
     <layout class="QVBoxLayout" name="verticalLayout12_2">
      <item>
       <widget class="QLabel" name="Pokemon26">
        <property name="styleSheet">
         <string notr="true">font: 11pt &quot;Fixedsys&quot;;
color: rgb(0, 0, 0);
</string>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="CheckButton26">
        <property name="styleSheet">
         <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0.972, y2:0.909, stop:0 rgba(220, 0, 0, 253), stop:0.301676 rgba(147, 0, 0, 254), stop:0.653631 rgba(131, 0, 0, 254), stop:1 rgba(0, 0, 0, 255));

font: 11pt &quot;Fixedsys&quot;;
border-color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>Check</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="3">
     <layout class="QVBoxLayout" name="verticalLayout4_2">
      <item>
       <widget class="QLabel" name="Pokemon18">
        <property name="styleSheet">
         <string notr="true">font: 11pt &quot;Fixedsys&quot;;
color: rgb(0, 0, 0);
</string>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="CheckButton18">
        <property name="styleSheet">
         <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0.972, y2:0.909, stop:0 rgba(220, 0, 0, 253), stop:0.301676 rgba(147, 0, 0, 254), stop:0.653631 rgba(131, 0, 0, 254), stop:1 rgba(0, 0, 0, 255));

font: 11pt &quot;Fixedsys&quot;;
border-color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>Check</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="5" column="1">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="Pokemon27">
        <property name="styleSheet">
         <string notr="true">font: 11pt &quot;Fixedsys&quot;;
color: rgb(0, 0, 0);
</string>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="CheckButton27">
        <property name="styleSheet">
         <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0.972, y2:0.909, stop:0 rgba(220, 0, 0, 253), stop:0.301676 rgba(147, 0, 0, 254), stop:0.653631 rgba(131, 0, 0, 254), stop:1 rgba(0, 0, 0, 255));

font: 11pt &quot;Fixedsys&quot;;
border-color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>Check</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="5" column="2">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="Pokemon28">
        <property name="styleSheet">
         <string notr="true">font: 11pt &quot;Fixedsys&quot;;
color: rgb(0, 0, 0);
</string>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="CheckButton28">
        <property name="styleSheet">
         <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0.972, y2:0.909, stop:0 rgba(220, 0, 0, 253), stop:0.301676 rgba(147, 0, 0, 254), stop:0.653631 rgba(131, 0, 0, 254), stop:1 rgba(0, 0, 0, 255));

font: 11pt &quot;Fixedsys&quot;;
border-color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>Check</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="background">
   <property name="geometry">
    <rect>
     <x>-440</x>
     <y>-140</y>
     <width>1791</width>
     <height>991</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resource.qrc">:/resources/resources/pokedexBackground.jpg</pixmap>
   </property>
  </widget>
  <zorder>background</zorder>
  <zorder>gridLayoutWidget</zorder>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
